src/pages/collection-details/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/png" href="/images/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../style.css">
+    <link rel="stylesheet" href="./collection-details.css">
+    <script type="module" src="../../main.js"></script>
+    <title>Film Book</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,700;0,900;1,100;1,300;1,700;1,900&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header-component></header-component>
+    <main class="main-content gray-section">
+        <div class="collection">
+            <div class="collection__top-details">
+                <div class="collection__images">
```

Comment:
wiesz co myślę że wystarczy tutaj 1 zdjęcie które opisuje całą kolekcję. Obecnie to niewiele widać z tych 6 obrazków

Reply:
Ok. Sugerowałem się naszym prototypem. Zmienione na jedno zdjęcie.